pkg/3scale/amp/operator/apicast_options_provider.go

```diff
@@ -35,7 +35,7 @@ func (a *ApicastOptionsProvider) GetApicastOptions() (*component.ApicastOptions,
 	a.apicastOptions.ImageTag = product.ThreescaleRelease
 	a.apicastOptions.OpenSSLVerify = strconv.FormatBool(*a.apimanager.Spec.Apicast.OpenSSLVerify)
 	a.apicastOptions.ResponseCodes = strconv.FormatBool(*a.apimanager.Spec.Apicast.IncludeResponseCodes)
-	boolTmp := a.apimanager.IsMonitoringEnabled()
+	boolTmp := true
```

Comment:
Does this change have any impact? Why has this been changed to always be true and not dependant on the monitoring cr flag?

Reply:
For simplicity, it will be always enabled for operator deployments and disabled for template deployments.